app/src/main/java/com/example/memeitupapp/ui/gridmemes/GridMemesFragment.kt

```diff
@@ -4,39 +4,68 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.GridLayoutManager
 import com.example.memeitupapp.R
 import com.example.memeitupapp.data.entity.Meme
-import com.example.memeitupapp.util.DEFAULT_INT
+import com.example.memeitupapp.ui.adapter.GridMemesAdapter
 import com.example.memeitupapp.util.Data
 import com.example.memeitupapp.util.Event
-import kotlinx.android.synthetic.main.fragment_grid_memes.activity_main_call_service_button
-import kotlinx.android.synthetic.main.fragment_grid_memes.activity_main_first_meme_name
+import com.example.memeitupapp.util.Status
+import kotlinx.android.synthetic.main.fragment_grid_memes.fragment_grid_memes_progress_bar
+import kotlinx.android.synthetic.main.fragment_grid_memes.fragment_grid_memes_recycler_view
 
 class GridMemesFragment : Fragment() {
 
     private lateinit var gridMemesViewModel: GridMemesViewModel
+    private lateinit var gridMemesAdapter: GridMemesAdapter
```

Comment:
you just need to use this inside showMemes?

Reply:
removed to the function that use it, thanks